src/Form/AdvancedSettingsForm.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+namespace Drupal\shareaholic\Form;
+
+use Drupal\Core\Cache\CacheBackendInterface;
+use Drupal\Core\Config\Config;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\shareaholic\Api\ShareaholicApi;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Class AdvancedSettingsForm.
+ */
+class AdvancedSettingsForm extends ConfigFormBase {
+
+  /** @var @var ShareaholicApi */
+  private $shareaholicApi;
+
+  /** @var Config */
+  private $shareaholicConfig;
+
+  public function __construct(ConfigFactoryInterface $config_factory, ShareaholicApi $shareaholicApi, Config $config)
+  {
+    parent::__construct($config_factory);
+    $this->shareaholicApi = $shareaholicApi;
+    $this->shareaholicConfig = $config;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get('config.factory'),
+      $container->get('shareaholic.api'),
+      $container->get('shareaholic.editable_config')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'shareaholic_advanced_settings_form';
+  }
+
+  /**
+     * {@inheritdoc}
+     */
+    protected function getEditableConfigNames() {
+      return [];
+    }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+
+    $form['advanced'] = [
+      '#type' => 'details',
+      '#open' => TRUE,
+      '#title' => $this->t('Advanced'),
+    ];
+
+    $form['advanced']['disable_og_tags'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Disable Open Graph tags (it is recommended NOT to disable open graph tags)'),
```

Comment:
Can we actually flip these to be:

```
[ CHECKBOX ] Include Open Graph Tags
```
The negative statement can be confusing.

Reply:
Does it imply that we has to also flip the OG per-node setting?

Reply:
We can leave it as-is for per-node for now.